cypress/e2e/misdemeanor/marijuana/pages/m-offense-mari-1-cont.cy.ts

```diff
@@ -0,0 +1,25 @@
+describe('m-offense-mari-1-cont', () => {
+  it('have all necessary information', () => {
+    cy.visit('/calculator/m-offense-mari-1-cont');
+
+    cy.get('[data-cy="calc-block-of-content"]')
+      .should('have.length', 1)
+      .should('be.visible');
+
+    cy.get('a:visible')
+      .filter((_index, element: any) => element.getAttribute('href')
```

Comment:
hey @LeoFuna I have a suggestion to improve the `any` typing here. Here's the [gist](https://gist.github.com/sramalho94/533e30bc628f6c051611bf32f17016b0). I've tested it locally and it passed but let me know if you experience any issues!

Reply:
Great catch Stephan! When i was coding this test i've typed as `any` and i've just forgot to adjust